staff/views/repos.py

```diff
@@ -204,7 +204,13 @@ def build_workspace(workspace, all_users):
         )
         workspaces = [build_workspace(w, users) for w in workspaces]
 
+        def build_contact(user):
+            return f"{user.fullname} <{user.email}>" if user.fullname else user.email
```

Comment:
In other places we're using `user.notifications_email` to send emails to users, would it make sense to use that here too if the main purpose is to use it for contacting users?